src/E1_coldflow/config.h

```diff
@@ -71,6 +72,9 @@ namespace config {
     sensors[1] = {"All Pressure",  FLIGHT_BRAIN_ADDR, 1, 1};
     sensors[2] = {"Battery Stats", FLIGHT_BRAIN_ADDR, 2, 3};
     sensors[3] = {"Aux temp",      FLIGHT_BRAIN_ADDR, 4, 1};
+    sensors[5] = {"Number Packets Sent", FLIGHT_BRAIN_ADDR, 5, 1};
```

Comment:
make the last number here 10, and add the packet id and details to this sheet https://docs.google.com/spreadsheets/d/1xNmjrb5a28FfVg-MG9jUBRiPyfdkjo7HiqPWP41N_IQ/edit#gid=998950668 in the `packet-id` tab